pkg/controller/jenkins/configuration/base/resources/pod.go

```diff
@@ -79,6 +79,16 @@ func GetJenkinsMasterContainerBaseEnvs(jenkins *v1alpha2.Jenkins) []corev1.EnvVa
 	return envVars
 }
 
+// GetJenkinsHomePath fetches the Home Path for Jenkins
+func GetJenkinsHomePath(jenkins *v1alpha2.Jenkins) string {
+	for _, envVar := range jenkins.Spec.Master.Containers[0].Env {
+		if envVar.Name == "JENKINS_HOME" {
+			return envVar.Value
```

Comment:
do we need to sanitize this input form envs? like, trim whitespace

Reply:
No we don't need to AFAIK